packages/core/src/database/seed.ts

```diff
@@ -79,6 +80,9 @@ export const createDatabaseCli = async (dsn: string) => {
       await pool.query(sql`${raw(query)}`);
       console.log(`${chalk.blue('[create-tables]')} Run ${file} succeeded.`);
     }
+
+    await updateDatabaseTimestampToLatest(pool);
```

Comment:
or just `updateDatabaseAlterationTimestamp`? we can add a timestamp param later, and if no param is specified then use the latest

Reply:
updated